.github/labeler.yml

```diff
@@ -0,0 +1,74 @@
+# Based on paths provided in the CODEOWNERS file
+
+# Add 'awaiting-review' to any PR opened (starting with just team-ExternalDeps)
+awaiting-review:
+- src/main/java/com/google/devtools/build/lib/bazel/bzlmod/**/*
+- src/test/java/com/google/devtools/build/lib/bazel/bzlmod/**/*
+- src/test/py/bazel/bzlmod/**/*
+# - '**/*'
+
+team-ExternalDeps:
+- src/main/java/com/google/devtools/build/lib/bazel/bzlmod/**/*
```

Comment:
Also `src/main/java/com/google/devtools/build/lib/bazel/repository/` and `src/test/java/com/google/devtools/build/lib/bazel/repository/`
/cc @Wyverald Any other directories should be included here?

Reply:
Thanks! I've added these 2.